craft_parts/plugins/rust_plugin.py

```diff
@@ -37,7 +37,7 @@
 # see https://github.com/samuelcolvin/pydantic/issues/975#issuecomment-551147305
 # The proper fix requires Python 3.9+ (needs `typing.Annotated`)
 if TYPE_CHECKING:
```

Comment:
GitHub's not letting me apply the suggestion to the full thing, but:

```suggestion
UniqueStrList Annotated[liststr], conlist(str, unique_items=True)]
```